app/controllers/parent/kids_controller.rb

```diff
@@ -12,12 +12,16 @@ def new
   def create
     @kid = Kid.new(kid_params)
     @kid.parents << Parent.find_by_user_id(current_user.id)
+    current_date = Date.today
+    age = ((current_date - @kid.birthdate).to_i)/365
```

Comment:
Yeah this is the second time seeing this. Make a helper function so you don't have to rewrite code